data/data2.s

```diff
@@ -2690,10 +2690,7 @@ gUnknown_082162F8:: @ 82162F8
 	.incbin "baserom.gba", 0x002162f8, 0x8
 
 gUnknown_08216300:: @ 8216300
-	.incbin "baserom.gba", 0x00216300, 0x3
-
-gUnknown_08216303:: @ 8216303
```

Comment:
Why should this incbin be combined with the previous one? It's not clear they really belong together.